src/builder.rs

```diff
@@ -556,6 +561,50 @@ impl<P: fmt::Debug, V> Bundle<InProgress<P, PartiallyAuthorized>, V> {
     }
 }
 
+impl<P: fmt::Debug, V> Bundle<InProgress<P, PartiallyAuthorized>, V> {
```

Comment:
These are two adjacent `impl` blocks with the same bounds, so they can be merged:
```suggestion

```